acceptance-tests/src/test/scala/ch/renku/acceptancetests/workflows/Collaboration.scala

```diff
@@ -62,4 +65,47 @@ trait Collaboration {
     projectPage.Collaboration.Issues.NewIssue.createIssueButton click;
     sleep(3 seconds)
   }
+
+  def verifyBranchWasAdded(implicit projectDetails: ProjectDetails): Unit = {
+    val projectPage = ProjectPage()
+    When("the user navigates to the Collaboration tab")
+    click on projectPage.Collaboration.tab
+    Then("they should see a 'Do you want to create a merge request for branch...' banner")
```

Comment:
A tiny, tiny thing. Probably that should read `Then("he should see...` (or she :)

Reply:
"they" can be used this way as a gender-neutral singular pronoun

Reply:
Let us leave this as "they".